pkg/devfile/adapters/common/utils.go

```diff
@@ -175,9 +176,15 @@ func GetVolumes(devfileObj devfileParser.DevfileObj) map[string][]DevfileVolume
 				}
 			}
 
+			// if there is no volume mount path, default to volume mount name as per devfile schema
+			mountPath := filepath.Join(string(os.PathSeparator), volumeMount.Name)
```

Comment:
Since most openshift containers are linux machines, wont this create create `\<volume-name>` when running odo from a windows machine?

Reply:
`os.PathSeparator` would take care about underlying OS

Reply:
Yes and that would create`\<volume-name`on windows, which seems invalid to me

Reply:
ya right, we should default use `/` as separator.

Reply:
updated, pls help re-review the PR, thx!